README.md

```diff
@@ -51,6 +49,7 @@ This will display:
     mjr:~/work/node_redis (master)$
 
 Note that the API is entirely asynchronous. To get data back from the server, you'll need to use a callback.
+From v.2.6 on the API supports camelCase and snack_case and all options / variables / events etc. can be used either way.
```

Comment:
Sorry for the confusion, I totally agree that options, events etc etc shouldn't be passed as uppercase. I was only referring to the method names / redis commands. Probably should have read the whole readme before leaving this comment